.eslintrc.json

```
@@ -38,7 +43,7 @@
             "rules": {
                 "no-var": "error",
                 "prefer-const": "error",
-                "prettier/prettier": "warn"
+                "prefer-arrow-callback": "warn"
```

```suggestion
                "prettier/prettier": "warn",
                "prefer-arrow-callback": "warn"
```

proposing to keep this on warn level to not change the appearance of style errors compared to the past months. would not have a functional impact but style issues would be marked as errors otherwise. Personally I want to reserve errors for actual issues in the code.